packages/fscomponents/src/components/ModalHalfScreen.tsx

```diff
@@ -19,11 +20,11 @@ const styles = StyleSheet.create({
     bottom: -400
   }
 });
-
 export interface ModalHalfScreenProps {
   visible?: boolean;
   height?: number;
   onRequestClose: () => void;
+  verticalPosition?: string;
```

Comment:
This should be defined as a enum with values top, middle, bottom: https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
I should have worked harder implementing that - Thanks for the learning on how to use enums @bweissbart.  I also added a small library for calculating the StatusBar height when using 'top' position.